howso/engine/session.py

```diff
@@ -184,22 +190,22 @@ def set_metadata(self, metadata: Optional[Dict[str, Any]]) -> None:
         self._metadata = metadata
         self._update()
 
-    def _update_attributes(self, session: BaseSession) -> None:
+    def _update_attributes(self, session) -> None:
```

Comment:
```suggestion
    def _update_attributes(self, session: dict) -> None:
```

Reply:
Should be addressed by: d66654ea10038f70a275bb4aa504128552745e60

Reply:
Changes still missing in latest commit